examples/tutorials/advanced/cartesian_histograms.py

```diff
@@ -0,0 +1,338 @@
+r"""
```

Comment:
It's very likely that the leading `r` is not needed:
```suggestion
"""
```